File: extras/enum/enum.py
import os
import sys

from enum import Enum

class TEMP(Enum):
    HOT = 1
    TEMPERATE = 2
    COLD = 3


def main():
        
    feeling = TEMP.HOT

    if feeling == TEMP.HOT:
        print("I'm feeling hot")

    return 0


if __name__ == "__main__":

    sys.exit(main())

File: extras/enum/enum.cpp
#include <iostream>

using namespace std;

int main()
{
    enum TEMP {HOT, TEMPERATE, COLD};
    
    TEMP feeling {TEMP::HOT};
    
    if (feeling == TEMP::HOT)
    {
        std::cout << "I'm feeling hot!";
    }

    return 0;
}
